src/node/rpc/member_frontend.h

```diff
@@ -1349,7 +1349,7 @@ namespace ccf
         };
```

Comment:
I can't make a suggestion on the previous line, but I believe it should be:
```
return make_success(ma->state_digest);
```

This has changed the reported schema but not the value we actually return, which is still a `MemberAck` rather than a `StateDigest`.

Reply:
Yes, thank you, for some reason I thought the json wrapper would error out, but that doesn't make sense.